KsApi/Service.swift

```diff
@@ -308,7 +308,10 @@ public struct Service: ServiceType {
       .flatMap(ErroredBackingsEnvelope.producer(from:))
   }
 
-  public func fetchManagePledgeViewBacking(id: Int, withStoredCards: Bool)
+  /**
+   FIXME: Reconsider the naming here as it returns both the `Project` and the `Backing` object.
```

Comment:
Since it's just us two here, feel free to take the lead on renaming things as you see fit.